lvgl_tft/ssd1306.c

```diff
@@ -153,96 +148,92 @@ void ssd1306_set_px_cb(lv_disp_drv_t * disp_drv, uint8_t * buf, lv_coord_t buf_w
 
 void ssd1306_flush(lv_disp_drv_t * disp_drv, const lv_area_t * area, lv_color_t * color_p)
 {
-    uint8_t row1 = 0, row2 = 0;
-   	i2c_cmd_handle_t cmd;
-
-    // Divide by 8
-    row1 = area->y1 >> 3;
-    row2 = area->y2 >> 3;
-
-	cmd = i2c_cmd_link_create();
-	i2c_master_start(cmd);
-	i2c_master_write_byte(cmd, (OLED_I2C_ADDRESS << 1) | I2C_MASTER_WRITE, true);
-
-	i2c_master_write_byte(cmd, OLED_CONTROL_BYTE_CMD_STREAM, true);
-	i2c_master_write_byte(cmd, OLED_CMD_SET_MEMORY_ADDR_MODE, true);
-	i2c_master_write_byte(cmd, 0x00, true);
-	i2c_master_write_byte(cmd, OLED_CMD_SET_COLUMN_RANGE, true);
-	i2c_master_write_byte(cmd, area->x1, true);
-	i2c_master_write_byte(cmd, area->x2, true);
-	i2c_master_write_byte(cmd, OLED_CMD_SET_PAGE_RANGE, true);
-	i2c_master_write_byte(cmd, row1, true);
-	i2c_master_write_byte(cmd, row2, true);
-	i2c_master_stop(cmd);
-	i2c_master_cmd_begin(I2C_NUM_0, cmd, 10/portTICK_PERIOD_MS);
-	i2c_cmd_link_delete(cmd);
-
-	cmd = i2c_cmd_link_create();
-	i2c_master_start(cmd);
-	i2c_master_write_byte(cmd, (OLED_I2C_ADDRESS << 1) | I2C_MASTER_WRITE, true);
-
-	i2c_master_write_byte(cmd, OLED_CONTROL_BYTE_DATA_STREAM, true);
-	i2c_master_write(cmd, (uint8_t *)color_p, OLED_COLUMNS * (1+row2-row1), true);
-	i2c_master_stop(cmd);
-	i2c_master_cmd_begin(I2C_NUM_0, cmd, 10/portTICK_PERIOD_MS);
-	i2c_cmd_link_delete(cmd);
+    /* Divide by 8 */
+    uint8_t row1 = area->y1 >> 3;
+    uint8_t row2 = area->y2 >> 3;
+
+    uint8_t conf[] = {
+        OLED_CONTROL_BYTE_CMD_STREAM,
+        OLED_CMD_SET_MEMORY_ADDR_MODE,
+        0x00,
+        OLED_CMD_SET_COLUMN_RANGE,
+        (uint8_t) area->x1,
+        (uint8_t) area->x2,
+        OLED_CMD_SET_PAGE_RANGE,
+        row1,
+        row2,
+    };
+
+    send_bytes(disp_drv, conf, sizeof conf);
```

Comment:
Is it valid this way? Shouldn't be `sizeof(conf)`?

Reply:
`sizeof` is valid without parentheses if used on expressions, according to [Stack Overflow](https://stackoverflow.com/a/28075117).

Reply:
I can add the parentesi can add the parentheses, according to the link they doesn't hurt.

Reply:
Fixed in 9cdc350

Reply:
Thanks! :)